src/test/java/org/opensearch/neuralsearch/processor/TextImageEmbeddingProcessorIT.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.neuralsearch.processor;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+
+import lombok.SneakyThrows;
+
+import org.apache.http.HttpHeaders;
+import org.apache.http.message.BasicHeader;
+import org.apache.http.util.EntityUtils;
+import org.junit.After;
+import org.opensearch.client.Response;
+import org.opensearch.common.xcontent.XContentHelper;
+import org.opensearch.common.xcontent.XContentType;
+import org.opensearch.neuralsearch.common.BaseNeuralSearchIT;
+
+import com.google.common.collect.ImmutableList;
+
+/**
+ * Testing text_and_image_embedding ingest processor. We can only test text in integ tests, none of pre-built models
```

Comment:
How are we confirming that this works?

Reply:
We have tested that offline by setting up one of [BedRock titan](https://aws.amazon.com/bedrock/titan/) models and registering it using [remote connector feature](https://opensearch.org/docs/latest/ml-commons-plugin/extensibility/connectors/) in ml-commons.